sdk/storage/Azure.Storage.Common/src/Shared/StorageAudience.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Storage
+{
+    /// <summary>
+    /// Audience for retrieving and AAD token.
+    /// </summary>
+    public readonly struct StorageAudience : IEquatable<StorageAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="StorageAudience"/>.
+        /// </summary>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="value"/> is null.
+        /// </exception>
+        public StorageAudience(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string StorageValue = "Storage";
```

Comment:
The value of the extended enum should be `"https://storage.azure.com/.default"` rather than `"Storage"`.  It's okay if the property name/string value aren't a one to one match.  This will eliminate the mapping code you have in `BlobClientOptionsExtensions.AsPolicy` and make it possible for someone to add their own string value if some weird environment/config supports a new value in the future.  See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ContainerRegistryAudience.cs for a similar example.

Reply:
Fixed!